lib/app_styles.dart

```diff
@@ -371,3 +371,15 @@ const darkListTileTheme =
 
 // New Onboarding Screen Colors
 const lightOnboardingScreen = Color.fromARGB(255, 245, 240, 228);
+
```

Comment:
Adding a note to remove unused grayscaleInvertMatrix & darkLogoShadowColor after the upcoming release